dreammall-earth/dreammall.earth
# build(deps): revert vike from 0.4.181 to 0.4.176

..in /presenter (#1492)

Motivation
----------
The docker image was broken after the update. A revert of the commit fixes the issue.

It's pretty annoying that we cannot catch this during CI (at the moment).

How to test
-----------
1. `docker compose -f docker-compose.yml build presenter`
2. `docker compose -f docker-compose.yml up presenter`
3. Visit http://localhost:3001/
4. On `master` you wil see "Cannot GET /" with the following error:
```
presenter-1  | Error: [@brillout/vite-plugin-server-entry@0.4.5][Wrong Usage] Cannot find server entry. (Re-)build your app and try again. If you still get this error, then you may need to manually import the server entry, see https://github.com/brillout/vite-plugin-server-entry#manual-import
presenter-1  |     at importServerEntry (/app/node_modules/@brillout/vite-plugin-server-entry/dist/importServerEntry/index.js:51:29)
presenter-1  |     at async loadImportBuild (/app/node_modules/vike/dist/cjs/node/runtime/globalContext/loadImportBuild.js:15:9)
presenter-1  |     at async initGlobalContext (/app/node_modules/vike/dist/cjs/node/runtime/globalContext.js:111:30)
presenter-1  |     at async renderPageAndPrepare (/app/node_modules/vike/dist/cjs/node/runtime/renderPage.js:70:9)
presenter-1  |     at async renderPage_wrapper (/app/node_modules/vike/dist/cjs/node/runtime/renderPage.js:30:24)
presenter-1  |     at async renderPage (/app/node_modules/vike/dist/cjs/node/runtime/renderPage.js:50:35)
presenter-1  |     at async /app/build/index.cjs:2:1257
```

This reverts commit 87f0f2283a48059795219ee6fc50c3c5b27fa8d7.
